### ISSUE TITLE
jshell is not documented in accumulo command help message.

### ISSUE THREAD

**Issue by keith-turner** [apache/accumulo#3052]
**Describe the bug**

The help for the accumulo command does not document jshell.

```
$ ./accumulo

Usage: accumulo <command> [--help] (<argument> ...)

  --help   Prints usage for specified command

Core Commands:
  admin                          Executes administrative commands
  ec-admin                       Executes administrative commands for external compactions
  help                           Prints usage
  info                           Prints Accumulo cluster info
  init                           Initializes Accumulo
  shell                          Runs Accumulo shell
  version                        Prints Accumulo version
  classpath                      Prints Accumulo classpath
  <main class> args              Runs Java <main class> located on Accumulo classpath

Process Commands:
  compaction-coordinator         Starts Accumulo Compaction Coordinator
  compactor                      Starts Accumulo Compactor
  gc                             Starts Accumulo garbage collector
  manager                        Starts Accumulo Manager
  master                         Starts Accumulo master (Deprecated)
  minicluster                    Starts Accumulo minicluster
  monitor                        Starts Accumulo monitor
  sserver                        Starts Accumulo scan server
  tserver                        Starts Accumulo tablet server
  zookeeper                      Starts Apache Zookeeper instance

Other Commands:
  check-compaction-config        Verifies compaction config within a given file
  check-server-config            Checks server config
  config-upgrade                 converts properties store in ZooKeeper to 2.1 format
  convert-config                 Convert Accumulo configuration from XML to properties
  create-empty                   Creates an empty rfile
  create-token                   Creates authentication token
  dump-zoo                       Writes Zookeeper data as human readable or XML to a file.
  generate-splits                Generate split points from a set of 1 or more rfiles
  login-info                     Prints Accumulo login info
  rfile-info                     Prints rfile info
  split-large                    Splits an RFile into large and small key/value files
  wal-info                       Prints WAL Info
  zoo-info-viewer                view Accumulo instance and property information stored in ZooKeeper
  zoo-zap                        Utility for zapping Zookeeper locks
```

The following can be run even though its not documented in the help.

```
$ ./accumulo jshell
Preparing JShell for Apache Accumulo

Use 'client' to interact with Accumulo

|  Welcome to JShell -- Version 11.0.16
|  For an introduction type: /help intro

jshell> 
```

**Expected behavior**

The help documents jshell.

**Comment by AlbertWhitlock** [comment]
Where should jshell be documented in this list.......under 'Core commands'?

**Comment by keith-turner** [comment]
> Where should jshell be documented in this list.......under 'Core commands'?

Core commands seems like a good place to place it.

**Comment by AlbertWhitlock** [comment]
PR #3100 addresses this ticket.

**Comment by jmark99** [comment]
Closing issue as this was resolved in  #3100 as mentioned above.
